tools/install.sh

```diff
@@ -5,17 +5,49 @@ if [[ $(id -u) -ne 0 ]] ; then echo "Please run the installation script as root"
 SCRIPT_DIR="$( cd "$(dirname "$0")" ; pwd -P )"
 CURRENT_PADIR="$(dirname "$SCRIPT_DIR")"
 PA_FOLDER_NAME="$(basename "$CURRENT_PADIR")"
+OLD_PADIR=
+OS=
+if which yum > /dev/null 2>&1; then
+   OS="RedHat"
+elif which apt-get > /dev/null 2>&1; then
+   OS="Debian"
+fi
```

Comment:
I think that a _else_ block should be added to stop the execution of the script (`exit`) but also to inform the user that its OS is not supported by this installation script.